### DISCUSSION TITLE
Is raspap compatible with cloudconnexa?

### DISCUSSION THREAD

**Discussion by ksirichai** [RaspAP/raspap-webgui#1712] [Q&A]
I want to connect to cloudconnexa but I receive an error message on the dashboard of cloudconnexa saying "The Client does not support web authentication."
What should I do?

**Comment by billz** [comment]
You're likely using CloudConnexa SSO web authentication. This requires an additional auth step in the browser that RaspAP is unable to negotiate:

<img src="https://github.com/user-attachments/assets/f17cee01-c1ee-4b5f-909a-1d060772129e" width="300"  />

Instead of this method, you can use [username / password authentication](https://openvpn.net/cloud-docs/owner/settings/settings---user-authentication/set-cloudconnexa--username-password--authentication-for-users.html).
